README.md

```diff
@@ -1,11 +1,11 @@
-# docker-eXist (WIP)
+# docker-eXist 
 minimal exist-db docker image with FO support
 
-[![Build Status](https://travis-ci.org/duncdrum/exist-docker.svg?branch=master)](https://travis-ci.org/duncdrum/exist-docker)
-[![Codacy Badge](https://api.codacy.com/project/badge/Grade/ace7cb88e9934b5f9ae772e981db177f)](https://www.codacy.com/app/duncdrum/exist-docker?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=duncdrum/exist-docker&amp;utm_campaign=Badge_Grade)
-[![](https://images.microbadger.com/badges/image/duncdrum/exist-docker.svg)](https://microbadger.com/images/duncdrum/exist-docker "Get your own image badge on microbadger.com")
-[![](https://images.microbadger.com/badges/version/duncdrum/exist-docker.svg)](https://microbadger.com/images/duncdrum/exist-docker "Get your own version badge on microbadger.com")
-[![](https://images.microbadger.com/badges/commit/duncdrum/exist-docker.svg)](https://microbadger.com/images/duncdrum/exist-docker "Get your own commit badge on microbadger.com")
+[![Build Status](https://travis-ci.com/eXist-db/docker-existdb.svg?branch=master)](https://travis-ci.com/eXist-db/docker-existdb)
+[![Codacy Badge](https://api.codacy.com/project/badge/Grade/ace7cb88e9934b5f9ae772e981db177f)](https://www.codacy.com/app/eXist-db/docker-existdb?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=eXist-db/docker-existdb&amp;utm_campaign=Badge_Grade)
+[![](https://images.microbadger.com/badges/image/existdb/existdb.svg)](https://microbadger.com/images/existdb/existdb "Get your own image badge on microbadger.com")
```

Comment:
Why is this badge and then next 2 badges pointing at existdb/existdb? There is no GitHub repo at that address

Reply:
Ah... is that the Docker hub address?

Reply:
yes i m not 100% which needs to point at which, but we ll only see after we published the first images

Reply:
Ok